src/NorthwindWebsite/API/Controllers/ProductsController.cs

```diff
@@ -19,7 +19,7 @@ public ProductsController(IProductService productService)
     [ProducesResponseType(StatusCodes.Status200OK)]
     public async Task<ActionResult<List<Product>>> GetProducts()
     {
-        var products = await _productService.GetAll();
+        var products = await _productService.GetSimpleProducts();
```

Comment:
What it the the **key** difference between `GetProducts` and `GetSimpleProducts`? Looking at repository, actually, I cannot find it. Let's revert this change

Reply:
@dotnet-fizzyy  The difference is that in GetProducts we return a number of objects by reference. For example, in Product class we get Category by reference. As the result, it creates enormous jsons, that cannot be displayed by a browser or Postman, or processed in code. Basically what I do by returning simplified Products representation - I do not include Category or Supplier objects in them. 
For the task I need only ProductName or CategoryName. If I needed to include, for example, Supplier or Category names, I would need to create a separate dto to extract only the name from Category or Supplier and pass them to Products.

Reply:
Ok then. So, for the future, you may resolve situation with including/excluding related entities using the following cases:
1. Pass some query param like `includeRelatives`;
2. Create a separate endpoint to return lightweight models. This option is quite rare and should have some serious changes comparing to casual endpoint;